packages/internal/src/paths.ts

```diff
@@ -18,6 +18,7 @@ const PATH_WEB_DIR_LAYOUTS = 'web/src/layouts/'
 const PATH_WEB_DIR_PAGES = 'web/src/pages/'
 const PATH_WEB_DIR_COMPONENTS = 'web/src/components'
 const PATH_WEB_DIR_SRC = 'web/src'
+const PATH_WEB_WEBPACK_CONFIG = 'web/webpack.config.js'
```

Comment:
It should be called `PATH_WEB_DIR_CONFIG` so that it matches the naming convention of the other files.

Reply:
> Let's place this inside of a `config` directory: `web/config/webpack.config.js`
> 
> I imagine that we would like to have a single place to store configs.

What about `web/.babelrc.js`, `web/jsconfig.json`? Those are config files too, so to keep it consistent they should also be moved to `web/config` directory. 

I also use `postcss.config.js` - I don't know if it would be autodetected inside `web/config`.

My suggestion is to keep it in `web` directory but ofc you have the last word :)

Reply:
I feel you, but `.babelrc.js` and `js|tsconfig.js` have to be in those locations to work. If I could move them I would!

It looks like `postcss.config.js` can be specified: https://github.com/postcss/postcss-loader#config

We're likely going to add configs for jest, and jest's environment setup soon and it'll be nice to not have the root directory cluttered with a bunch of configuration files.

Reply:
Done.